src/serializers/html/html.ts

```diff
@@ -135,6 +142,28 @@ function createHTMLSerializer(schema: Schema): HTMLSerializerReturnType {
     }
 }
 
-export { createHTMLSerializer, INITIAL_MARKED_OPTIONS }
+/**
+ * [[DESCRIPTION]]
+ */
+const htmlSerializerInstanceById: HTMLSerializerInstanceById = {}
+
+/**
+ * [[DESCRIPTION]]
+ *
+ * @param id [[DESCRIPTION]]
+ * @param schema [[DESCRIPTION]]
+ *
+ * @returns [[DESCRIPTION]]
+ */
+// TODO: Needs unit tests!
+function getHTMLSerializerInstance(id: NonEmptyString, schema: Schema) {
```

Comment:
💭 Given that the `TypistEditor.id` prop is set to `default` in case the user doesn't specify one, it could make sense to switch the parameters and make `id` optional (defaulting to `default` too).